Src/Plugins/Security/SEC101_006.GitHubPatValidator.cs

```diff
@@ -57,6 +57,19 @@ public static ValidationState IsValidDynamic(ref Fingerprint fingerprint, ref st
                 return ValidationState.NoMatch;
             }
 
+            if (groups.TryGetNonEmptyValue("checksum", out string checksum))
```

Comment:
>checksum [](start = 44, length = 8)

why aren't we implementing this right now?

Reply:
Was talking with Eddy about it and since it isn't trivial, he asked that I get in the new regex and then come back later to use the checksum.  It only reduces the number of false positives.  

---
In reply to: [608156928](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608156928) [](ancestors = 608156928)

Reply:
ok.

---
In reply to: [608175867](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608175867) [](ancestors = 608175867,608156928)

Reply:
As discussed, we'll do this in V2

---
In reply to: [608187463](https://github.com/microsoft/sarif-pattern-matcher/pull/366#discussion_r608187463) [](ancestors = 608187463,608175867,608156928)